core/src/main/java/hudson/cli/CLICommand.java

```diff
@@ -242,70 +242,72 @@ public int main(List<String> args, Locale locale, InputStream stdin, PrintStream
         this.locale = locale;
         CmdLineParser p = getCmdLineParser();
 
+        CliContext context = new CliContext(getName(), args, getTransportAuthentication2());
+
         // add options from the authenticator
         SecurityContext sc = null;
         Authentication old = null;
-        Authentication auth;
         try {
             // TODO as in CLIRegisterer this may be doing too much work
             sc = SecurityContextHolder.getContext();
             old = sc.getAuthentication();
 
-            sc.setAuthentication(auth = getTransportAuthentication2());
+            sc.setAuthentication(getTransportAuthentication2());
```

Comment:
Not: Could still assign `#getTransportAuthentication` to a variable instead of calling it twice.